op-node/rollup/derive/attributes.go

```diff
@@ -57,7 +59,9 @@ func (ba *FetchingAttributesBuilder) PreparePayloadAttributes(ctx context.Contex
 	// case we need to fetch all transaction receipts from the L1 origin block so we can scan for
 	// user deposits.
 	if l2Parent.L1Origin.Number != epoch.Number {
+		start := time.Now()
 		info, receipts, err := ba.l1.FetchReceipts(ctx, epoch.Hash)
+		log.Debug("FetchingAttributesBuilder:PreparePayloadAttributes: called ba.l1.FetchReceipts.", "duration", time.Since(start))
```

Comment:
I recommend using key/value pairs to make this a bit more readable/searchable in the logs. Something like:

```suggestion
		log.Debug("derivation fetcher performance info", "stage", "FetchingAttribtuesBuilder", "caller", "PreparePayloadAttributes", "method", "FetchReceipts", "duration", time.Since(start))`
```